cmd/thanos/main.go

```diff
@@ -40,6 +41,11 @@ const (
 type setupFunc func(*run.Group, log.Logger, *prometheus.Registry, opentracing.Tracer) error
 
 func main() {
+	if os.Getenv("DEBUG") != "" {
```

Comment:
why not flag? ):

Reply:
Uhm, yea, historically used to it I suppose. We can change it later :)
Flags need a revisit before we cut some v0.1 or so anyway.